examples/internal/clients/generateunboundmethods/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# Folders
+_obj
+_test
+
+# Architecture specific extensions/prefixes
+*.[568vq]
+[568vq].out
+
+*.cgo1.go
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+*.exe
+*.test
+*.prof
```

Comment:
I think this file is generated by `swagger-codegen` if there isn't one already, and it looks like we're using a simpler version in our other generated clients: https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/clients/abe/.gitignore. Maybe we could rewrite this one too?